src/Uno.UI/UI/Xaml/Controls/ListViewBase/ListViewBase.cs

```diff
@@ -609,8 +609,8 @@ internal override void OnItemClicked(int clickedIndex, VirtualKeyModifiers modif
 
 			void SingleSelectionCase()
 			{
-
-				if (ItemsSource is ICollectionView collectionView)
+				//Verifying if the ItemsSource is a CollectionView and not ISelectionInfo
```

Comment:
We should be explaining here _why_ we added this check, not necessarily explaining what the code is doing here.

Reply:
No further explanation has been added here yet